contracts/ethereum/l1Aggregator.sol

```diff
@@ -13,13 +13,15 @@ contract L1Aggregator {
     //TODO: an asset can be present on multiple Starknet bridges,
     //so approving all bridges where the token is collateralized should be considered in the future
     mapping(address => address) public bridges;
```

Comment:
let's update `bridges` to support multiple addresses: `mapping(address => address[]) bridges;  `